src/aten/SoftMax.cpp

```diff
@@ -0,0 +1,72 @@
+#include <ATen/ATen.h>
+#include <ATen/XPUNativeFunctions.h>
+#include <ATen/core/Tensor.h>
+
+namespace at {
+
+// Following three functions are currently not use by sdpa
```

Comment:
Please check, if stock CUDA registers, we register.

Reply:
Even our highest priority is HF/TIMM/TB, we are gradually providing on-par op coverage as CUDA. If you are implementing an operator, you need register and implement its variants as well. It won't take your a lot efforts.